src/content/h1/engine/renderer/readme.md

```diff
@@ -24,11 +41,14 @@ The renderer needed to be adapted for the range of user hardware for the PC port
 * The [fog][] screen layers effect for simulated volumetric fog does not render at all.
 * HUD shield meters are missing their flash effect when drained.
 * Monochrome bitmaps and p8 bump map formats are unsupported.
+* Custom Edition skips rendering some effects when they are not loaded yet, such as the engine lens flares in a10's intro or initial bullet hole decals.
```

Comment:
This is done to avoid stutters btw

Reply:
If they were in predicted resources would this not be the case?

Reply:
Afaik predicted resources don't always nest far enough